### ISSUE TITLE
403 Forbidden (Sabre\DAV\Exception\Forbidden).

### ISSUE THREAD

**Issue by TheToto318** [nextcloud/server#26992]
Hey 

So i get a forbidden message when I try to upload in local external storage. Witch is a bind mount in an lxc container (previligied) in proxmox. The www-data user and group are the owners of the mounted folder (same uid and gid as the ones on the host). I can write to the folder in cli with the www-data user.

I managed to get the upload working by setting the particular permission to the folder (0755). Now everything is working, EXECPT writting in the root of the folder, I can write into the subfolder of this folder but not into the folder himself.

I can't also delete folder into the mounted folder but can erase subdirectories into these folder.

Regards

### Steps to reproduce
1. Create a bind mount in proxmox lxc.
2. Give the www-data user full acces of this mount 
3. Try to upload a file into the root of the folder.

### Expected behaviour
Upload successfuly

### Actual behaviour
"Forbidden" message

### Server configuration

**Operating system:** Debian 9

**Web server:** Apache 

**Database:** Mysql

**PHP version:** 7.4

**Nextcloud version:** 21.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install

</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
External storage
Talk
Onlyoffice
Calendar
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
<?php
$CONFIG = array (
  'instanceid' => 'ocp1doe25f5g',
  'passwordsalt' => '',
  'secret' => '',
  'trusted_domains' => 
  array (
    0 => '',
  ),
  'datadirectory' => '/var/www/nextcloud-data',
  'dbtype' => 'mysql',
  'version' => '21.0.1.1',
  'overwrite.cli.url' => '',
  'htaccess.RewriteBase' => '/',
  'dbname' => 'nextcloud',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
  'mysql.utf8mb4' => true,
  'dbuser' => '',
  'dbpassword' => '',
  'installed' => true,
  'memcache.local' => '\OC\Memcache\Redis',
  'memcache.distributed' => '\OC\Memcache\Redis',
```
</details>

**Are you using external storage, if yes which one:** Local

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
No log about the issue
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No log about this specific issue except into the nextcloud sync client "Sabre\DAV\Exception\Forbidden"
```
</details>

**Comment by antiuser** [comment]
I can confirm this issue.
When i create odt file in shared directory and then do "Save As" from Collabora Online i get a message with error
![Снимок экрана_2021-05-18_14-41-49](https://user-images.githubusercontent.com/5499424/118630215-c4ae0780-b7e7-11eb-862e-e61de7a4015d.png)
![Снимок экрана_2021-05-18_14-42-33](https://user-images.githubusercontent.com/5499424/118630279-d2638d00-b7e7-11eb-8909-2ee6ed4742fc.png)
![Снимок экрана_2021-05-18_14-43-30](https://user-images.githubusercontent.com/5499424/118630300-d5f71400-b7e7-11eb-9fb0-1c6776dcc1f1.png)

But "Save" function works as usual

**Comment by TheToto318** [comment]
No workaround for this ?

**Comment by DanScharon** [comment]
@antiuser your issues seem to be fixed by https://github.com/nextcloud/richdocuments/pull/1559 and  https://github.com/nextcloud/richdocuments/pull/1550/ so you should wait until the next update of the Collabora App

**Comment by antiuser** [comment]
@DanScharon 
> @antiuser your issues seem to be fixed by [nextcloud/richdocuments#1559](https://github.com/nextcloud/richdocuments/pull/1559) and [nextcloud/richdocuments#1550](https://github.com/nextcloud/richdocuments/pull/1550) so you should wait until the next update of the Collabora App

Yeah, im upgrade NC to latest version and now working as expected :) thank you DEVs :)

**Comment by TheToto318** [comment]
Tried to update to the last NC version, for me I have the same problem... 
Anybody have the correct permission settings for local external storage in NC ?
